plugins/flowdock/lib/samson_flowdock/samson_plugin.rb

```diff
@@ -20,22 +21,47 @@ def flowdock_tokens
     def no_flowdock_token?(flowdock_attrs)
       flowdock_attrs['token'].blank?
     end
+
+    def enabled_flows_names
+      flowdock_flows.enabled.map(&:name)
+    end
+  end
+end
+
+Samson::Hooks.callback :deploy_defined do
+  Deploy.class_eval do
```

Comment:
@fneves, I'm pretty sure this extends classes - copy and paste this into `irb`:

```ruby
class Oracle
  def are_you_there?
    p 'yes'
  end
end

class Oracle
  def are_you_here?
    p 'yes'
  end
end

oracle = Oracle.new
oracle.are_you_there?
oracle.are_you_here?
```

Not 100% sure how it will work with inheritance but I've seen this pattern used